pkg/providers/instancetype/instancetype.go

```diff
@@ -104,10 +104,6 @@ func (p *DefaultProvider) List(ctx context.Context, kc *corev1beta1.KubeletConfi
 	if kc == nil {
```

Comment:
Should we just remove this and always pass a kubleteConfiguration now?

Reply:
It can still be nil. We just pass it transparently, so it's possible someone hasn't set it.